templates/zerver/dev_tools.html

```diff
@@ -0,0 +1,40 @@
+{% extends "zerver/portico.html" %}
+
+{# Login page. #}
+{% block portico_content %}
+<div id="devtools-page">
+    <h2>Useful development URLs</h2>
+    <p>
+        Below is a list of useful URLs that provide information useful in development.
```

Comment:
Not sure if this is too obvious, but maybe it could be worth mentioning that the development server must be running to be able to see these?

I don't know if it may be confusing for people e.g. running only `./tools/test-js-with-node --coverage` and having problems when going directly to http://localhost:9991/node-coverage/index.html.